src/openmrs-esm-error-handling.ts

```diff
@@ -14,3 +14,45 @@ window.onerror = function() {
 
   return false;
 };
+
+window.onunhandledrejection = function(event) {
+  showToast({ description: `UNHANDLED PROMISE REJECTION: ${event.reason}` });
```

Comment:
The all caps `UNHANDLED PROMISE REJECTION:` is a very poor user experience. Toasts are shown to real users - not to developers. The message in them should be something a user can understand.

```suggestion
  showToast({ description: `Oops! An unexpected error occurred.` });
```